UnityProject/Assets/Mirror/Runtime/NetworkServer.cs

```diff
@@ -1541,6 +1538,8 @@ static void BroadcastToConnections()
 		            //Connection disconnected
 		            if (connection.isReady == false) continue;
 
+		            if (identity.observers.ContainsKey(connection.connectionId) == false) continue;
```

Comment:
I think you could just shortcut this with a bool that gets Set once the client has loaded, tho When we do network bubbling We will require this